includes/rest/class-interaction.php

```diff
@@ -43,6 +44,22 @@ public static function register_routes() {
 				),
 			)
 		);
+
+		\register_rest_route(
+			ACTIVITYPUB_REST_NAMESPACE,
+			'/reactions/(?P<id>\d+)',
```

Comment:
This reads like the ID is the ID of a reaction, but these are the reactions for this ID, so maybe we should change it to something like `/post/(?P<id>\d+)/reaction` instead.

That would be also more consistent with the comments and other posts endpoints:

* https://github.com/Automattic/wordpress-activitypub/blob/253cffff12291ae2ae40538ae16fefc44b69e9a3/includes/rest/class-comment.php#L38
* https://github.com/Automattic/wordpress-activitypub/blob/253cffff12291ae2ae40538ae16fefc44b69e9a3/includes/rest/class-collection.php#L81

In this context, perhaps we can also create a dedicated `class-post.php` file, to encapsulate the post endpoints.